cmd/flags.go

```diff
@@ -37,4 +38,10 @@ var (
 			"Example: <protocol>://<ip>:<port>, tcp://127.0.0.1:26657",
 		DefValue: "",
 	}
+	mutualPeersFlag = &flag.Flag{
+		Name: "p2p.mutual",
+		Usage: "Comma-separated Multiaddresses of mutual peers to keep unbreakable connection with. " +
```

Comment:
Is the multiaddress format documented somewhere?
```suggestion
		Usage: "Comma-separated multiaddresses of mutual peers to keep an unbreakable connection with. " +
```

Reply:
https://multiformats.io/multiaddr/

Reply:
or http://multiformats.io.ipns.localhost:8080/multiaddr/ for cool people ;)

Reply:
Is it possible to put a link to that in the docstring?

Reply:
I think so. This is a string that is shown to you once you do `--help`.